apps/client/src/theme/ontimeTab.ts

```diff
@@ -1,19 +1,25 @@
-export const ontimeTab = {
-  tab: {
-    fontWeight: 600,
-    borderBottom: '2px solid transparent',
-    color: '#9d9d9d', // $gray-500
-    marginBottom: '-2px',
-    _selected: {
-      color: '#101010', // $ui-black
-      border: 'none',
-      borderBottom: '2px solid #779BE7', // $blue-400
+import { defineSlotRecipe } from '@chakra-ui/react';
+import { tabsAnatomy } from '@chakra-ui/react/anatomy';
+
+export const ontimeTabRecipe = defineSlotRecipe({
```

Comment:
question: looks like this can be removed?

Reply:
Yup. Removed in https://github.com/cpvalente/ontime/pull/1438/commits/805599526e95b28b61cce13962ae732a5690b71a